### DISCUSSION TITLE
Deleted repository shown in taskbar

### DISCUSSION THREAD

**Discussion by RobertoBM21** [community/community#77875] [New to GitHub]
I have left a repository but it still can be seen in the taskbar. When I try to leave from settings it does not appear though. The repository was generated by Github Education Program so I do not have the settings option inside the repository. Its name is series-intro-to-github-flow and the creator appear as Exp-Intro-to-GitHub-Flow-Cohort-1. 

If anyone could tell me how to solve this situation I would appreciate.

**Comment by queenofcorgis** [comment]
Hi there - did you clone this repository, star it, and/or watch it?

**Reply by RobertoBM21** [reply]
It was generated automatically by github education. So I could not delete it but leave it.

**Reply by celesteamador** [reply]
I have the same problem

**Reply by celesteamador** [reply]
@RyanHecht @JordiEspinozaMendoza
